a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -1048,6 +1048,7 @@ class GetBackupVThunder(VThunderBaseTask):
     @axapi_client_decorator
     def execute(self, vthunder):
         try:
+            time.sleep(20)
```

Comment:
We can't make sure how long takes and we may also don't need this wait. So 
- can we reference VThunderComputeConnectivityWait() to use a loop to retry and default we don't sleep.
- please also take care GetMasterVThunder()

Reply:
I'll handle this problem by another bug, since we got several bugs related to this sleep time.